dev/tests/acceptance/tests/functional/Magento/FunctionalTest/Catalog/Section/StorefrontProductInfoMainSection.xml

```diff
@@ -51,5 +51,7 @@
         <element name="productAddToCompare" type="button" selector="a.action.tocompare"/>
         <element name="productOptionDropDownTitle" type="text" selector="//label[contains(.,'{{var1}}')]" parameterized="true"/>
         <element name="productOptionDropDownOptionTitle" type="text" selector="//label[contains(.,'{{var1}}')]/../div[@class='control']//select//option[contains(.,'{{var2}}')]" parameterized="true"/>
+
+        <element name="productSwatchText" type="text" selector="//div[@class='swatch-option text' and contains(text(),'{{var1}}')]" parameterized="true"/>
```

Comment:
Please rename `var1` to something meaningful. For example `{{swatchOptionText}}`